incubator/command-config/src/test/resources/io/aklivity/zilla/runtime/command/config/internal/asyncapi/mqtt/proxy/complete/zilla.yaml

```diff
@@ -0,0 +1,74 @@
+name: example
+bindings:
+  tcp_server0:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port:
+      - 1883
+      - 8883
+    routes:
+    - exit: mqtt_server0
+      when:
+      - port: 1883
+    - exit: tls_server0
+      when:
+      - port: 8883
+  tls_server0:
+    vault: server
+    type: tls
+    kind: server
+    options:
+      keys:
+      - "${{env.TLS_SERVER_KEY}}"
+      sni:
+      - "${{env.TLS_SERVER_SNI}}"
+      alpn:
+      - "${{env.TLS_SERVER_ALPN}}"
+    exit: mqtt_server0
+  mqtt_server0:
+    type: mqtt
+    kind: server
+    routes:
+    - exit: mqtt_client0
+      when:
+      - topic: smartylighting/streetlights/1/0/event/*/lighting/measured
```

Comment:
This is probably need to be changed to accomodate the new mqtt configs

Reply:
done